framework/src/node/node.ts

```diff
@@ -507,6 +507,7 @@ export class Node {
 					...this._options.genesisConfig,
 				},
 				registeredModules: this.getRegisteredModules(),
+				wsPort: this._options.network.port,
```

Comment:
i would prefer to have 
```
{
  port: this._options.network.port
}
```
so that we can add more network information if required, and it is consistent with the config structure

Reply:
Do you mean

1. 
```ts
{
  ...
  network: {
    port: port: this._options.network.port
  }
  ...
}
```

2.
```ts
{
  ...
  port: port: this._options.network.port
  ...
}
```

Pick one 😄 
I like the 1. better, it was suggested by @nazarhussain on discord. It might feel empty right now but at least there is context. One may imagine having a chain augmenting the nodeInfo that would conflict in the absence of a namespace for network props.

Reply:
ah sorry i meant option `1`!